include/swift/Frontend/ArgsToFrontendOutputsConverter.h

```diff
@@ -147,6 +148,14 @@ class SupplementaryOutputPathsComputer {
   Optional<std::vector<SupplementaryOutputPaths>>
   readSupplementaryOutputFileMap() const;
 
+  /// The driver should have created an entry for each primary input, whether
+  /// or not there are any supplementary outputs. If any are missing, something
+  /// has gone wrong.
+  /// Diagnose the missing entries.
+  /// The diagnostics will stop the compilation.
```

Comment:
Nitpick: paragraph breaks mean something in Doxygen (they separate the summary from the full description), but only if you do two of them. If you don't want that, please don't wrap before the end of the line.

I also don't think it's worth mentioning that the diagnostics will stop compilation. That's not something ArgsToFrontendOutputsConverter needs to think about.

Reply:
The whole comment is gone, now.